includes/_footer.htm

```diff
@@ -32,7 +32,8 @@
 				'streams',
 				'quickstart',
 				'toc',
-				'upgrade'
+				'upgrade',
+				'content'
```

Comment:
I did this on purpose so we could have a generic name for containers that we know aren't going to rendered alongside themselves. It just makes it easier for other to copy and paste existing pages and build off of them.